server/usecases/account/user.go

```diff
@@ -65,6 +67,10 @@ func FindOrCreateUserFromGithub(token string) (*domain.User, error) {
 	return domain.FindOrCreateFromGithub(githubUser, token, primaryEmail, InjectUserRepository())
 }
 
+func UserProjects(u *domain.User) ([]*project.Project, error) {
```

Comment:
__[Golint]__ exported function UserProjects should have comment or be unexported ([link](https://sider.review/gh/repos/50192141/pulls/454#issue-3810550))

<sub>posted by Sider</sub>